tests/server/amo/TestDetailsPage.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-loop-func */
+import config from 'config';
+import request from 'supertest-as-promised';
+
+import { runServer } from 'core/server/base';
+
+
+const fetchMock = require('fetch-mock');
+
+const defaultURL = '/en-US/firefox/addon/fakeaddon/';
+const detailsAPIURL = `${config.get('apiHost')}/api/v3/addons/addon/fakeaddon/?lang=en-US`;
+
+describe('Details Page', () => {
+  let app;
+
+  before(() => runServer({ listen: false, app: 'amo' })
```

Comment:
I'm guessing the `webpackIsomorphicTools.undo()` is required because of this? Could there be a helper like:

```js
function setupServer(app, cb) {
  before(() => runserver({ listen: false, app })
    .then(cb)
  );

  after(() => webpackIsomorphicTools.undo());
}

describe('Details Page', () => {
  let app;

  setupServer((server) => { app = server; });
});